composer.json

```diff
@@ -12,10 +12,10 @@
     }
   ],
   "require": {
-    "php-http/httplug-bundle": "^1.7",
-    "php-http/curl-client": "^1.7",
+    "php-http/httplug-bundle": "^2.0",
```

Comment:
No need for this

Reply:
dunno why i keep thinking we bumped the major version here... fixed.